examples/tutorials/advanced/dataset_on_surface.py

```diff
@@ -0,0 +1,167 @@
+"""
```

Comment:
This page (http://vterrain.org/Misc/draping.html) says:

> The process of projecting a 2D feature on a 3D surface is known as "draping".

Perhaps rename the file to `draping_on_3d_surface.py`?

Reply:
Renamed in https://github.com/GenericMappingTools/pygmt/pull/3316/commits/b3da9994d5f4b6111b1e47c4a6cce31b782a84a0.